src/frontend/src/index.ts

```diff
@@ -10,9 +10,26 @@ import { recoveryWizard } from "./flows/recovery/recoveryWizard";
 import { showWarningIfNecessary } from "./banner";
 import authorizeAuthentication from "./flows/authenticate";
 
+/**
+ * Shows some version information, retrieved from /version
+ */
+const showVersion = async () => {
+  const response = await fetch("/version");
```

Comment:
This won't work in practice on mainnet. The `/version` has to be certified otherwise the service worker will reject the payload. Since asset hashes are also recalculated on upgrade and install, just certifying the version there as well should not be a problem, right?
(Note: just using raw does not help, as the service worker will require certification for calls to raw on pages loaded on non-raw domains.)

If you use `agent-js` to make the query call instead of using fetch, you could also bypass the certification requirement. Not a huge fan of that approach though.

Alternatively, we could just not print the version in the front-end code. And if somebody is interested in what the version is, they could just manually visit https://identity.raw.ic0.app/version (i.e. the same approach as for /metrics).

Reply:
I think I'll just bake in the version in the assets, that was my first idea but our asset types are a bit silly. Will give it another try.